app/controllers/defendants_controller.rb

```diff
@@ -85,4 +73,8 @@ def set_unlink_attempt
                         UnlinkAttempt.new
                       end
   end
+
+  def defendant_identifier
+    @defendant.arrest_summons_number || @defendant.national_insurance_number
```

Comment:
can this be `defendant.arrest_summons_number || defendant.national_insurance_number` and could it be moved to DefendantHelper concern?

Reply:
👍 changed to defendant.arrest_summons_number || defendant.national_insurance_number and could it be moved to DefendantHelper concern.